### PR TITLE
fix: more typos in the code

### PR DESCRIPTION
**Pull request by szepeviktor**
Continuation of #4448

Found few more typos.

### CODE REVIEW

File: packages/ee/ui-platform/src/lib/components/cards/configure-confederated-authn-card/configure-confederated-authn-card.component.ts [hunk 1]
```diff
@@ -62,7 +62,7 @@ export class ConfigureConfederatedAuthnCardComponent
 
     if (this.federatedAuthnProvider === 'Google') {
       delete platform.federatedAuthProviders.google;
-    } else if (this.federatedAuthnProvider === 'Github') {
+    } else if (this.federatedAuthnProvider === 'GitHub') {
```
Comment (szepeviktor) [inline packages/ee/ui-platform/src/lib/components/cards/configure-confederated-authn-card/configure-confederated-authn-card.component.ts:65]: Should I revert these?

File: packages/server/api/src/app/ee/referrals/referral.entity.ts [hunk 2]
```diff
@@ -12,7 +12,7 @@ export type ReferralSchema = Referral & {
 }
 
 export const ReferralEntity = new EntitySchema<ReferralSchema>({
-    name: 'referal',
+    name: 'referral',
```
Comment (abuaboud) [inline packages/server/api/src/app/ee/referrals/referral.entity.ts]: Please revert this, as it's table name

File: packages/server/api/test/integration/cloud/authn/cloud-authn.test.ts [hunk 3]
```diff
@@ -303,7 +303,7 @@ describe('Authentication API', () => {
             const responseBody = response?.json()
 
             const referral = await databaseConnection
-                .getRepository('referal')
+                .getRepository('referral')
```
Comment (abuaboud) [inline packages/server/api/test/integration/cloud/authn/cloud-authn.test.ts]: Same for here

File: packages/ee/ui-platform/src/lib/components/cards/configure-confederated-authn-card/configure-confederated-authn-card.component.ts (hunk 1 above)
Comment (abuaboud) [reply packages/ee/ui-platform/src/lib/components/cards/configure-confederated-authn-card/configure-confederated-authn-card.component.ts:65]: Nope that's fine

**Review by abuaboud — CHANGES_REQUESTED** [review]
Thank you so much!

File: packages/server/api/test/integration/cloud/authn/cloud-authn.test.ts (hunk 3 above)
Comment (szepeviktor) [reply packages/server/api/test/integration/cloud/authn/cloud-authn.test.ts]: Done 🍏

**Comment by abuaboud** [comment]
@allcontributors please add @szepeviktor for code

**Review by abuaboud — DISMISSED** [review]
Awesome Thank you!

**Comment by szepeviktor** [comment]
Nx fails because typos has a problem: is excludes URL-like string.
Appearance helper was renamed in #4448 but the reference to it.

**Pull request merged**